src/taipy/core/_orchestrator/_orchestrator.py

```diff
@@ -222,7 +225,6 @@ def _on_status_change(cls, job: Job):
         if job.is_completed() or job.is_skipped():
             cls.__unblock_jobs()
         elif job.is_failed():
-            print(f"\nJob {job.id} failed, abandoning subsequent jobs.\n")
```

Comment:
Can we keep this message?
Or log it instead of `print`?

Or when the job is failed, we log a message saying why the job fails. It's better for the user, but probably quite tricky.

Reply:
We should definitely log (instead of `print`)

Reply:
It's a print I used for debugging previously that I forgot to remove

As for the logging issue, I think we need a different ticket for it as it might require more than a simple change @jrobinAV

Reply:
We have a log drped from the `job.update_status` method. Is it not enough?

Reply:
just had a look and I think it's sufficient